app/components/contributors/edit_component.html.erb

```diff
@@ -45,20 +45,20 @@
     </div>
     <div class="form-check">
       <%= form.radio_button :with_orcid, false, class: 'form-check-input', checked: false %>
```

Comment:
Extract radio button into a component?

Reply:
We have two uses of `form.radio_button` in the app now that look very different from one another, so I'd probably opt to leave that refactoring out of this work until the need/abstraction is clearer.